ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -632,24 +651,42 @@ module.exports = function OpenlayersMap(
       return this.createVectorLayer(locationModel, feature)
     },
     showMultiPolygonShape: function(locationModel) {
-      let lineObject = locationModel
+      const multiPolyObject = locationModel
         .get('multipolygon')
         .map(poly =>
           poly.map(coordinateSet =>
-            coordinateSet.map(pair => convertPointCoordinate(pair))
+            coordinateSet.map(pair => unconvertPointCoordinate(pair))
           )
         )
 
-      let feature = new Openlayers.Feature({
-        geometry: new Openlayers.geom.MultiPolygon(lineObject),
+      const bufferWidth =
+        DistanceUtils.getDistanceInMeters(
+          locationModel.get('polygonBufferWidth'),
+          locationModel.get('polygonBufferUnits')
+        ) || DEFAULT_BUFFER
+
+      const multiPolygon = Turf.multiPolygon(multiPolyObject)
+
+      const buffered = Turf.buffer(multiPolygon, bufferWidth, {
+        units: 'meters',
```

Comment:
According to Turf, only `miles, kilometers, degrees, or radians` are supported units.

Reply:
Their docs aren't fantastic they do support meters 
<img width="578" alt="screen shot 2019-01-07 at 10 43 27 am" src="https://user-images.githubusercontent.com/36056781/50783817-1c77f380-1269-11e9-9f88-5eda1b60b00a.png">

Reply:
Hmm, now I found another section in their docs that says `can be degrees, radians, miles, or kilometers inches, yards, metres, meters, kilometres, kilometers.` so now I'm not sure 😂

Reply:
According to their JSDocs for all their conversion methods they support `degrees, radians, miles, or kilometers inches, yards, metres, meters, kilometres, kilometers` as you said @adimka

Reply:
Looks like we use meters w/Turf in a bunch of other places in the codebase, FWIW.